teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```diff
@@ -104,6 +108,26 @@ public class VoluntaryExitCommand implements Runnable {
       arity = "1")
   private boolean confirmationEnabled = true;
 
+  @CommandLine.Mixin(name = "Data")
+  private ValidatorClientDataOptions dataOptions;
+
+  @CommandLine.Option(
+      names = {"--public-keys-filter"},
+      description = "Filter the exiting validators by public keys.",
```

Comment:
I would stick to `--validator-public-keys`. A filter is a technical thing IMO.
And I'd then specify better in the description what it does, something like:
`Optionally restrict the exit command to a specified list of public keys. When the parameter is not used, all keys will be exited`

Reply:
Done